module/platform/darwin/HardwareIO/src/Convert.h

```diff
@@ -37,9 +37,9 @@ namespace platform {
             /// This means 512 = 0
             static constexpr double ACCELEROMETER_CONVERSION_FACTOR = (4 * 9.80665) / 512.0;
 
-            /// The Gyrosocope value from the Darwin is between 0 and 1023, representing a value between -1600 and 1600
+            /// The Gyrosocope value from the Darwin is between 0 and 1023, representing a value between -500 and 500
```

Comment:
Did we ever verify these changes?

Reply:
We've tested the walk and it works, but no significant testing

Reply:
This is totally wrong lol